frontend/apps/variant-entity/src/components/cases/table-settings.tsx

```diff
@@ -153,10 +157,10 @@ function getOtherCasesColumns(t: TFunction<string, undefined>) {
     otherCasesColumnHelper.accessor(row => row.primary_condition_name, {
       id: 'primary_condition_name',
       cell: info => (
-        <div className="font-medium">
-          {info.getValue()}{' '}
-          <span className="font-mono text-xs text-muted-foreground">({info.row.original.primary_condition_id})</span>
-        </div>
+        <ConditionTableCell
+          conditionId={info.row.original.observed_phenotypes?.[0].id}
```

Comment:
use primary_condition_id and primary_condition_name (they are in uninterpreted cases as well)
phenotypes are not the same as conditions

Reply:
Also Phenotypes column is missing (using observed_phenotypes, display all the elements of the list in the cell, list can be empty)